README.md

```diff
@@ -282,23 +282,8 @@ You can use a custom property file that utilizes environment variables for many
 
 ```bash
 ./build-docker-image.sh && docker run -p 8080:8080 \
--e APIKEY=<apikey> \
```

Comment:
I removed most of these because the topic here is about using a custom application.yaml - not about specifying all of the environment settings.

@shubhamparikh927 Should we replace this with the steps needed to do it via docker-compose?